docs/API.md

```diff
@@ -1726,6 +1724,45 @@ const composePromise = minioClient.composeObject(destOption,sourceList)
 
 ```
 
+<a name="selectObjectContent"></a>
+### selectObjectContent(bucketName, objectName, selectOpts[, callback])
+
+Select contents of an object (S3 Select).
+
+__Parameters__
+
+| Param  |  Type | Description  |
+|---|---|---|
+| `bucketName`  |_string_   | Name of the bucket.  |
+| `objectName`  | _string_  | Name of the object.  |
+| `selOpts`  | _object_  |  |
```

Comment:
Including this one, there are still 2 more references to `selOpts`...
line #1569 in `src/test/unit/test.js`
line #3192 in `src/test/functional/functional-tests.js`

Reply:
Thank you so much.